### PR TITLE
Adding Multiplicity Distributions for Common Ancestors analysis

### PR DESCRIPTION
**Pull request by sartozza**

### CODE REVIEW

File: PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamBaseDump.cxx [hunk 1]
```diff
@@ -142,3 +142,5 @@ void AliFemtoDreamDump::SetEvent(std::vector<AliFemtoDreamBasePart> &vec,
     fOutputTree->Fill();
   }
 }
+
+
```
Comment (bhohlweger) [inline PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamBaseDump.cxx]: Remove

File: PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamCorrHists.h [hunk 2]
```diff
@@ -264,6 +264,14 @@ class AliFemtoDreamCorrHists {
...
   }
   ;
+    void FillSameEventMultDistCommon(int i, int iMult, float RelK) {
+    if (fSameEventMultDistCommon[i])
+      fSameEventMultDistCommon[i]->Fill(RelK, iMult);
+  }
+    void FillSameEventMultDistNonCommon(int i, int iMult, float RelK) {
+    if (fSameEventMultDistNonCommon[i])
+      fSameEventMultDistNonCommon[i]->Fill(RelK, iMult);
+  }
```
Comment (bhohlweger) [inline PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamCorrHists.h]: Check for !minimalbooking (also above in the fill same event dist non common)

File: PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamBaseDump.cxx (hunk 1 above)
Comment (sartozza) [reply PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamBaseDump.cxx]: done

File: PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamCorrHists.h (hunk 2 above)
Comment (sartozza) [reply PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamCorrHists.h]: done

**Comment by bhohlweger** [comment]
+1

**Pull request merged**